sandbox-backend/src/main/java/com/exadel/sandbox/team5/dto/CriteriaDto.java

```diff
@@ -0,0 +1,17 @@
+package com.exadel.sandbox.team5.dto;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.domain.Sort;
+
+@Getter
+@NoArgsConstructor
+public class CriteriaDto {
```

Comment:
`CriteriaDto` - it is too common name. I suggest to use for example `Sorting`

Reply:
done